@plotly/dash-generator-test-component-typescript/src/components/MemoTypeScriptComponent.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import {TypescriptComponentProps} from '../props';
+
+/**
+ * Description
+ * Example:
+ * ```
+ * @app.callback("clicks@btn")
+ * def on_click(*args):
+ *     return 1
+ * ```
```

Comment:
Funny docstring ;) maybe just a placeholder like "An example component using TypeScript and React.memo"? (here and in TypeScriptClassComponent)

Reply:
There is a test for the docstring containing `@` inside them, there is a checker function to get the comments, but it strips out the parts after a `@` because they are part of the documentation system, if you take the doc parts and join them, you lose the original indentation. The implementation also differs if a class component or a function component hence they both have the docstring.

For the fix, I did was to take the raw comments tokens col/row, strip out the `/*` parts and join them back.

Reply:
Got it, thanks :)